### PR TITLE
Changes for 6.3.1

### PR DESCRIPTION
**Pull request by THS-on**
Changes were tested on Debian 11, Ubuntu 20.4 (Focal) and Ubuntu 18.04 (Bionic).

On 18.04 the verifier event loop won't start with the packaged tornado version (4.5.3) but works fine when updated manually (see #11).

### CODE REVIEW

**Comment by THS-on** [comment]
@utkarsh2102 can you have a look at this PR?

**Comment by utkarsh2102** [comment]
Hey, please give me a few more days to review and merge this. Thank you! \o/

**Comment by THS-on** [comment]
@utkarsh2102 now with 6.3.2 released, should I change this PR to target 6.3.2 or do you want to review the changes for 6.3.1 first and I make a followup PR for 6.3.2?

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,14 @@
+keylime (6.3.1-1) unstable; urgency=medium
```
Comment (utkarsh2102) [inline debian/changelog:1]: Change this to UNRELEASED, please? :D

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+
+  [ Thore Sommer ]
```
Comment (utkarsh2102) [inline debian/changelog:3]: No need for this since you're the only contributor for this release. :D

File: debian/patches/series [hunk 3]
```diff
@@ -1 +1 @@
-embed-dataclasses-module.patch
+# embed-dataclasses-module.patch
```
Comment (utkarsh2102) [inline debian/patches/series:1]: Let's drop the patch then? :)

File: debian/python3-keylime-server.dirs [hunk 4]
```diff
@@ -1,2 +1,4 @@
 var/lib/keylime
 var/log/keylime
+var/run/keylime
+
```
Comment (utkarsh2102) [inline debian/python3-keylime-server.dirs:4]: Empty line ✂️

File: debian/python3-keylime-server.install [hunk 5]
```diff
@@ -1,9 +1,6 @@
 usr/bin/keylime_ca
 usr/bin/keylime_ima_emulator
 usr/bin/keylime_migrations_apply
-usr/bin/keylime_provider_platform_init
-usr/bin/keylime_provider_registrar
-usr/bin/keylime_provider_vtpm_add
```
Comment (utkarsh2102) [inline debian/python3-keylime-server.install:6]: I assume this is no longer provided?

**Review by utkarsh2102 — CHANGES_REQUESTED** [review]
Hellu,

Sorry for the delay, review is done. Please address these and I'll merge this right away and then let's work on the new release separately. Let me know if you have any questions or concerns.

File: debian/python3-keylime-server.install (hunk 5 above)
Comment (THS-on) [reply debian/python3-keylime-server.install:6]: Yes, they were no longer useful and removed in upstream

File: debian/changelog (hunk 2 above)
Comment (THS-on) [reply debian/changelog]: done

File: debian/changelog (hunk 1 above)
Comment (THS-on) [reply debian/changelog]: done

File: debian/patches/series (hunk 3 above)
Comment (THS-on) [reply debian/patches/series]: done

File: debian/python3-keylime-server.dirs (hunk 4 above)
Comment (THS-on) [reply debian/python3-keylime-server.dirs]: done

**Comment by utkarsh2102** [comment]
Thank you! \o/

**Comment by utkarsh2102** [comment]
I wonder if you can please create a branch in your fork, please? :D

**Comment by THS-on** [comment]
> I wonder if you can please create a branch in your fork, please? :D

I can do that. Should I still push the upstream import directly to this repo?

**Pull request merged**